sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-part-time-integration/e-cert-files/e-cert-file-record.ts

```diff
@@ -122,27 +122,27 @@ export class ECertPartTimeFileRecord extends ECertFileRecord {
   /**
    * Sum of all CSGP grants + BC Provincial Part-time grants from this certificate.
    */
-  totalGrantAmount: number;
+  totalCanadaGrantAmount: number;
   /**
    * Certificate number, the generated sequence number for the file.
    */
   certNumber: number;
   /**
    * BC Part-time grant amount 1.
```

Comment:
not sure if it`s part of the PR, I didn't get the `1` at the end. If it`s not making sense to devs, maybe we can remove it.